megamek/src/megamek/server/Server.java

```diff
@@ -25064,11 +24169,12 @@ private int damageExternalPassenger(Entity te, HitData hit, int damage,
      * @return <code>true</code> if the unit's engine exploded,
      * <code>false</code> if not.
      */
-    private boolean checkEngineExplosion(Entity en, Vector<Report> vDesc,
-            int hits) {
-        if(!(en instanceof Mech) && !(en instanceof QuadMech)
-            && !(en instanceof BipedMech) && !(en instanceof Aero)
-            && !(en instanceof Tank)) {
+    private boolean checkEngineExplosion(Entity en, Vector<Report> vDesc, int hits) {
+        // TODO : Windchild can I be simplified? - en instance of QuadMech is always false, as is
+        // TODO : en instanceof BipedMech according to my IDE
+        if (!(en instanceof Mech) && !(en instanceof QuadMech)
+                && !(en instanceof BipedMech) && !(en instanceof Aero)
+                && !(en instanceof Tank)) {
```

Comment:
Can I be simplified? - en instance of QuadMech is always false, as is  en instanceof BipedMech according to my IDE

Reply:
Yes, the quad and biped checks are redundant, as well as discriminatory toward TripodMech.

Reply:
There are Tripod Mechs?